src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public void getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/ab7af30b-ce15-4731-984c-57c1547b1f19)